pkg/services/accesscontrol/evaluator/evaluator_test.go

```diff
@@ -45,49 +50,54 @@ func TestEvaluatePermissions(t *testing.T) {
 		{
 			Name: "No expected scope always returns true",
 			HasScopes: map[string]struct{}{
-				"teams:*/permissions:*": {},
-				"users:*":               {},
-				"permissions:delegate":  {},
+				"teams:*":              {},
+				"permissions:*":        {},
+				"users:*":              {},
+				"permissions:delegate": {},
 			},
 			NeedAnyScope: []string{},
 			Valid:        true,
 		},
 		{
 			Name: "Single scope from  list",
 			HasScopes: map[string]struct{}{
-				"teams:1/permissions:delegate": {},
+				"teams:1":              {},
+				"permissions:delegate": {},
 			},
-			NeedAnyScope: []string{"teams:1/permissions:delegate"},
+			NeedAnyScope: []string{"teams:1", "permissions:delegate"},
 			Valid:        true,
 		},
 		{
 			Name: "Single scope from glob list",
 			HasScopes: map[string]struct{}{
-				"teams:*/permissions:*": {},
-				"users:*":               {},
-				"permissions:delegate":  {},
+				"teams:*":              {},
+				"permissions:*":        {},
+				"users:*":              {},
+				"permissions:delegate": {},
 			},
-			NeedAnyScope: []string{"teams:1/permissions:delegate"},
+			NeedAnyScope: []string{"teams:1", "permissions:delegate"},
 			Valid:        true,
 		},
 		{
 			Name: "Either of two scopes from glob list",
 			HasScopes: map[string]struct{}{
-				"teams:*/permissions:*": {},
-				"users:*":               {},
-				"permissions:delegate":  {},
+				"teams:*":              {},
+				"permissions:*":        {},
+				"users:*":              {},
+				"permissions:delegate": {},
 			},
 			NeedAnyScope: []string{"global:admin", "permissions:delegate"},
 			Valid:        true,
 		},
 		{
 			Name: "No match found",
 			HasScopes: map[string]struct{}{
-				"teams:*/permissions:*": {},
-				"users:*":               {},
-				"permissions:delegate":  {},
+				"teams:*":              {},
+				"permissions:*":        {},
+				"users:*":              {},
+				"permissions:delegate": {},
 			},
-			NeedAnyScope: []string{"teams1/permissions:delegate"},
+			NeedAnyScope: []string{"teams1", "permissions:nodelegate"},
```

Comment:
this test matches `permissions:nodelegate`, since the user has `permissions:*`.

Reply:
I need to review all the tests, there are a few that don't make sense after the change

Reply:
All of the test is rewritten in  [#35996](https://github.com/grafana/grafana/pull/35996)